lib/strainer/behaviors/action_view_image_tag_changes.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'action_view/helpers/asset_tag_helper'
+
+module Strainer
+  module Behaviors
+
+    class ActionViewImageTagChanges < Strainer::RuntimeBehavior
+      module HandleNilImageTagSrc
+        def image_tag(source, options={})
+          return tag("img", options) if source.nil?
```

Comment:
@dxypher This needs strainer_log for the case where source is nil. I don't think we can capture any additional useful context in the log.